### ISSUE TITLE
Summary!

### ISSUE THREAD

**Issue by tanelam** [heisrmann/pjs-capstone-lodash#7]
#### Score Level: 4
Comment(s): You did a great job on this project. You were able to get all the tests to pass! Your functions were well implemented and efficient. Make sure to use `const` when you can. 

#### Score Level: 4
Comment(s): Very nice work keeping everything neat and readable using consistent indentation, and comments throughout.

#### Score Level: 3
Comment(s): In general, great work getting everything set up. Your code is readable,  but some minor optimizations are still available. Great job taking advantage of the built-in methods throughout your program. 

### Overall Score: 11/12
Great work on this project! 👏😄

**Comment by heisrmann** [comment]
@tanelam ,
You are absolutely right about me overlooking the use of constant variables. I went back and reviewed where I could have used and updated by project locally.

I appreciate your generous feedback.  Enjoy this holiday season and the rest of the year.

Happy coding!
